src/actions/files.js

```diff
@@ -46,8 +47,13 @@ export function uploadFile(channelId, rootId, fileFormData, formBoundary) {
             files = await Client4.uploadFile(fileFormData, formBoundary);
         } catch (error) {
             forceLogoutIfNecessary(error, dispatch);
-            dispatch(batchActions([
-                {type: FileTypes.UPLOAD_FILES_FAILURE, error},
+            dispatch(batchActions([{
```

Comment:
Something is really funky with the indentation here. Do you mind making it clearer here which argument is for which function?